bouncer/shared/send_btc.ts

```diff
@@ -12,25 +12,33 @@ export const btcClient = new Client({
 });
 
 export async function selectInputs(amount: number) {
-  // List unspent UTXOs
-  const utxos = await btcClient.listUnspent();
+  return btcClientMutex.runExclusive(async () => {
+    // List unspent UTXOs
+    const utxos = await btcClient.listUnspent();
 
-  // Find a UTXO with enough funds
-  const utxo = utxos.find((u) => u.amount >= amount);
-  if (!utxo) throw new Error('Insufficient funds');
-  // TODO: be able to select more than one UTXO
+    // Find a UTXO with enough funds
+    const utxo = utxos.find((u) => u.amount >= amount);
+    if (!utxo) throw new Error('Insufficient funds');
```

Comment:
this introduces another issue right? If we have a single UTXO, one of the processes locks it, and then another process tries to send, and it gets an error because there are none there.

Reply:
Seems to be plenty available on a localnet. I tested with a bunch sending funds at the exact same time. We can solve that later if we ever hit the error.